developer/examples/build.vpc.catalogimage-offering.pkr.hcl

```diff
@@ -55,6 +55,9 @@ source "ibmcloud-vpc" "catalog" {
   vsi_interface      = "public"
   vsi_user_data_file = ""
 
+  vsi_boot_vol_capacity = "200"
```

Comment:
Just one example file is enough. You don't need to add this change to every example.